.github/workflows/ci.yml

```diff
@@ -10,6 +10,11 @@ on:
       - docs/sources/**
     branches:
       - main
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - labeled
```

Comment:
Was this being run for too many types previously?

Reply:
No, we're adding "labeled", but to do this we also need to include all defaults for "types"